presenters/keyword.go

```diff
@@ -1,13 +1,33 @@
 package presenters
 
 import (
+	"encoding/json"
+
 	"github.com/gutakk/go-google-scraper/models"
 )
 
 type KeywordPresenter struct {
 	Keyword models.Keyword
 }
 
+type KeywordResult struct {
+	NonAdwordLinks         []string
+	TopPositionAdwordLinks []string
+}
```

Comment:
Thanks for making the adjustment to simplify this struct. But I still think this struct can be removed altogether. I am not sure why you preferred to have a struct instead of two method like:

```
func (kp *KeywordPresenter) FormattedNonAdwordLinks() []string {
	var nonAdwordLinks []string
	return json.Unmarshal(kp.Keyword.NonAdwordLinks, &nonAdwordLinks)
}
```

In the view, it would prevent switching between `keyword` and `keywordResult`:

```
    {{ template "search_result" searchResultPartial "Total Non Adword Links" .data.keyword.NonAdwordsCount }}
    {{ template "search_result_list"  searchResultPartial "Non Adword Links" .data.keywordResult.NonAdwordLinks }}
```

What I recommend to is to simply pass the presenter only:

```
    {{ template "search_result" searchResultPartial "Total Non Adword Links" .data.keywordPresenter.keyword.NonAdwordsCount }}
    {{ template "search_result_list"  searchResultPartial "Non Adword Links" .data.keywordPresenter.FormattedNonAdwordLinks }}
```
```

Reply:
That is the good way to go but that solution lead to this [issue](https://github.com/gutakk/go-google-scraper/issues/46)

When I use like this
```html
{{ template "search_result_list"  searchResultPartial "Non Adword Links" .data.keywordPresenter.FormattedNonAdwordLinks }}
```
this is an error
```
executing "content" at <.data.keywordPresenter.FormattedTopPositionAdwordLinks>: can't evaluate field FormattedTopPositionAdwordLinks in type interface {}
```

After I researched on google and try this
```html
{{ with .data.keywordPresenter }}
    {{ template "search_result_list"  searchResultPartial "Top Position Adword Links" .FormattedTopPositionAdwordLinks }}
{{ end }}
```

and this is an error
```
executing "content" at <.FormattedTopPositionAdwordLinks>: can't evaluate field FormattedTopPositionAdwordLinks in type presenters.KeywordPresenter
```

I still can't find the solution how to solve this

Reply:
Noted. How about then using a more meaningful name:

```
type KeywordLinks struct {
    NonAdwordLinks         []string
	TopPositionAdwordLinks []string
}
```

`KeywordResult` is neither very clear or explicit.

Reply:
Fixed in 678cf1436697f08804df0e0f675fb80568a697cd